apps/buildog/src/app/organizations/[organizationId]/web/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import * as React from "react";
+import SimplicityHeader from "@/components/themes/simplicity-header";
+import TealHeader from "@/components/themes/teal-header";
+import CyanHeader from "@/components/themes/cyan-header";
+
+import { Button } from "@ui/components/button";
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "@ui/components/dropdown-menu";
+
+export default function DropdownMenuCheckboxes() {
+  const [header, setHeader] = React.useState<string>("1");
+
+  const test = (idx: string) => {
+    switch (idx) {
+      case "1":
+        return <SimplicityHeader />;
+      case "2":
+        return <TealHeader />;
+      case "3":
+        return <CyanHeader />;
+      default:
+        return null;
+    }
+  };
```

Comment:
Can you change the function name? and case names? we need to more understandable names